apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -50,7 +38,7 @@ router.get(
         assessment_id: res.locals.assessment.id,
         assessment_question_id: res.locals.assessment_question.id,
       },
-      InstanceQuestionRowSchema,
+      InstanceQuestionRowSchema.omit({ index: true }),
```

Comment:
I'm confused by this; AFAICT this is the only usage of `InstanceQuestionRowSchema`. Why is `index` needed at all?

Reply:
The index is computed a couple of rows below and used in the table to provide a stable but anonymous identifier for the instance question ("Instance 1", "Instance 2", etc.). This is different than the row index in the table itself, since sorting or filtering the list should keep these numbers stable. Anyway, I found a different way to compute this in the client side so this is not required, update coming.

Reply:
Done in b0ff010c9.